client/components/home/Pickup/card.tsx

```diff
@@ -0,0 +1,50 @@
+import { Text, Button, ButtonGroup, VStack, Heading } from "@chakra-ui/react";
+import { Card, CardBody } from "@chakra-ui/card";
+import { TSheetyPickupDetails } from "api/sheety/types";
+
+const OrgCard = ({
+	organisationName,
+	website,
+	categoriesAccepted,
+	time,
+	minimumWeightInKg,
+	pricingTermsInSgd,
+	contactMethod,
+	contactDetail,
+	lastUpdated
+}: TSheetyPickupDetails) => {
+	return (
+		<Card p={5} boxShadow={"lg"} direction={{ base: "column" }} variant={"elevated"}>
+			<CardBody>
+				<VStack spacing={"3"} mt={"6"}>
+					<Heading size={"md"} as={"b"}>
+						<Text>{organisationName}</Text>
+					</Heading>
+					<Text fontSize={"s"}>Please visit the website for more information.</Text>
+					<Text fontSize={"s"} as={"b"}>
+						You can donate:
+						<br />
+						<Text fontSize={"xs"}>
+							{categoriesAccepted}
+						</Text>
+					</Text>
+					<Text fontSize={"s"}>
+						<Text as={"b"}>Service Cost: {pricingTermsInSgd}</Text>
+						<br />
+						<Text as={"b"}>Min. Pickup Quantity: {minimumWeightInKg}</Text>
+					</Text>
+					<ButtonGroup>
+						<Button colorScheme={"teal"} variant={"outline"}>
```

Comment:
if the button function is to be implemented, can leave a comment as a TODO for the button.

i'm assuming that the data will come from the sheety data?

Reply:
implemented the webpage button, with the website link taken from sheety. no need to put todo.